src/js/components/loaders/dnaLoader.jsx

```diff
@@ -0,0 +1,41 @@
+"use strict";
```

Comment:
@Laludia its a component that I want to eventually display a loading animation. Currently it just has the string "Loading...". The idea was to have it be like an animation of a DNA strand hence the name, i will rename it so it's more clear though.